lib/kotkowo_web/components/pagination_bar.ex

```diff
@@ -0,0 +1,74 @@
+defmodule KotkowoWeb.Components.PaginationBar do
+  @moduledoc false
+
+  use Phoenix.Component
+
+  attr :selected_page, :integer, required: true
+  attr :last_page, :integer, required: true
+
+  def pagination_bar(assigns) do
+    assigns =
+      assigns
+      |> assign(:previous_page, assigns.selected_page - 1)
+      |> assign(:next_page, assigns.selected_page + 1)
+
+    ~H"""
+    <div class="flex flex-row text-xl gap-x-4 text-lg">
+      <button :if={@selected_page != 1} phx-click="select_page" class="p-4" value={@selected_page - 1}>
+        <img class="-scale-100" src="/images/arrow_right.svg" />
+      </button>
+      <img :if={@selected_page == 1} class="p-4" src="/images/arrow_left.svg" />
+      <div class="self-center gap-4 hidden lg:inline">
+        <button :if={@selected_page > 2} class="w-14 h-14" phx-click="select_page" value={1}>
```

Comment:
Don't include `phx-` tags in regular components.
Either move this to a LiveComponent or pass the `phx-` attributes from the parent.